Java/Pair.java

```diff
@@ -0,0 +1,14 @@
+package lfu;
```

Comment:
Try not to define common types like this on your own (as they will be prone to errors and re-work). There is a well known principle in programming called the DRY (DO NOT REPEAT YOURSELF).

Pairs are usually discouraged. Why not use an Auto_Value class here ?
This will also be a good learning for you.